.github/workflows/linting.yml

```diff
@@ -3,10 +3,7 @@ name: linting
 
 on: [push, pull_request]
 
-env:
-  TEMPLATE_REPOSITORY: T-Systems-MMS/terraform-template
-
 jobs:
   call-linting:
-    if: github.repository != '$TEMPLATE_REPOSITORY'
+    if: github.repository != 'T-Systems-MMS/terraform-template'
```

Comment:
maybe leave this check out and just disable actions for this repository? Or do we need to run some actions here?

Reply:
Change Actions policy to "Disable actions" setting for this repository.